runtime/src/weights/storage.rs

```diff
@@ -0,0 +1,199 @@
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 2.0.1
+
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::weights::{constants::RocksDbWeight as DbWeight, Weight};
+
+pub struct WeightInfo;
+impl storage::WeightInfo for WeightInfo {
+    fn delete_storage_bucket() -> Weight {
+        (355_000_000 as Weight)
+            .saturating_add(DbWeight::get().reads(3 as Weight))
+            .saturating_add(DbWeight::get().writes(1 as Weight))
+    }
+    fn update_uploading_blocked_status() -> Weight {
+        (208_000_000 as Weight)
+            .saturating_add(DbWeight::get().reads(2 as Weight))
+            .saturating_add(DbWeight::get().writes(1 as Weight))
+    }
+    fn update_data_size_fee() -> Weight {
+        (232_000_000 as Weight)
+            .saturating_add(DbWeight::get().reads(2 as Weight))
+            .saturating_add(DbWeight::get().writes(1 as Weight))
+    }
+    fn update_storage_buckets_per_bag_limit() -> Weight {
+        (217_000_000 as Weight)
+            .saturating_add(DbWeight::get().reads(2 as Weight))
+            .saturating_add(DbWeight::get().writes(1 as Weight))
+    }
+    fn update_storage_buckets_voucher_max_limits() -> Weight {
+        (256_000_000 as Weight)
+            .saturating_add(DbWeight::get().reads(2 as Weight))
+            .saturating_add(DbWeight::get().writes(2 as Weight))
+    }
+    fn update_number_of_storage_buckets_in_dynamic_bag_creation_policy() -> Weight {
+        (258_000_000 as Weight)
+            .saturating_add(DbWeight::get().reads(3 as Weight))
+            .saturating_add(DbWeight::get().writes(1 as Weight))
+    }
+    fn update_blacklist(i: u32, j: u32) -> Weight {
+        (463_903_000 as Weight)
+            .saturating_add((76_353_000 as Weight).saturating_mul(i as Weight))
+            .saturating_add((36_361_000 as Weight).saturating_mul(j as Weight))
+            .saturating_add(DbWeight::get().reads(203 as Weight))
```

Comment:
I'm not it's a good idea to change it manually at this point. It will be regenerated multiple times, runtime upgrade will happen, and we will change the hardware.

Reply:
I was wondering if we could tweak the benchmarking script in some way to fix this issue, but if that's not the case, perhaps you could make a GH issue, just so we don't forget about it?

This bug makes `update_blacklist` fee extremely high, especially w/ the blacklist size limit of 10000

Reply:
https://github.com/Joystream/joystream/issues/3517

Reply:
Actually, it appears that this problem is now solved after re-generating the weight functions :rocket: 
```
	fn update_blacklist(i: u32, j: u32, ) -> Weight {
		(0 as Weight)
			.saturating_add((58_286_000 as Weight).saturating_mul(i as Weight))
			.saturating_add((27_969_000 as Weight).saturating_mul(j as Weight))
			.saturating_add(DbWeight::get().reads(3 as Weight))
			.saturating_add(DbWeight::get().reads((1 as Weight).saturating_mul(i as Weight)))
			.saturating_add(DbWeight::get().reads((1 as Weight).saturating_mul(j as Weight)))
			.saturating_add(DbWeight::get().writes(1 as Weight))
			.saturating_add(DbWeight::get().writes((1 as Weight).saturating_mul(i as Weight)))
	}
```

Reply:
Ok!